Cakes-Co/index.html

```diff
@@ -1,16 +1,107 @@
+        <a class="nav__link" href="#">Home</a>
+        <a class="nav__link" href="#">Cakes</a>
+        <a class="nav__link" href="#">Ordering</a>
+        <a class="nav__link" href="#">Lessons</a>
+        <a class="nav__link" href="#">About</a>
+      </div>
+    </nav>
+
+    <section class="container">
+      <section class="content">
+        <article class="main-image">
+          <img
+            class="main-img"
+            src="https://images.unsplash.com/photo-1694168720027-5f2e02c59f14?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8OHx8c3dlZXQlMjBmcmVlJTIwaW1hZ2VzfGVufDB8fDB8fHww&auto=format&fit=crop&w=600&q=60"
+            alt="main img for page"
+          />
+        </article>
+
+        <article class="main">
+          <article class="welcome">
+            <h1>Welcome</h1>
+            <p>
+              Cakes hold an important place in all sorts of celebrations from
+              weddings to birthdays to graduations and work events. It seems no
+              celebration is complete without cake! As we all know, cakes aren’t
+              only enjoy at parties, but can be enjoyed for no reason at all.
+            </p>
+          </article>
+        </article>
+      </section>
+
+      <section class="responsive">
+        <section class="menu__img">
+          <div class="img-1">
+            <img
+              class="article__img"
+              src="https://img.freepik.com/free-photo/delicious-cake-candles-arrangement_23-2149023576.jpg?w=826&t=st=1696248234~exp=1696248834~hmac=7fa5baf28b9c6c9c36fbc0142d04b4fea30b6c921b45984e21d03b0edb747f72"
+              alt="more detail"
+            />
+          </div>
+
+          <div class="img-2">
+            <img
+              class="article__img"
+              src="https://images.unsplash.com/photo-1694168755444-001d2c137800?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=1974&q=80"
+              alt="more detail"
+            />
+          </div>
+
+          <div class="img-3">
+            <img
+              class="article__img"
+              src="https://ichef.bbci.co.uk/food/ic/food_16x9_1600/recipes/gluten-free_victoria_02196_16x9.jpg"
+              alt="you can enjoy your spare time."
+            />
+          </div>
 
-<body>
-    <!-- Add your markup here -->
-</body>
+          <div class="img-4">
+            <img
+              class="article__img"
+              src="https://cdn.pixabay.com/photo/2018/07/10/21/23/pancake-3529653_1280.jpg"
+              alt="you can visit anytime "
+            />
+          </div>
+        </section>
+      </section>
+    </section>
 
-</html>
\ No newline at end of file
+    <section class="responsive">
+      <footer class="footer">
+        <content class="footer__content">
+          <button>Phone</button>
+          <button>WhatsApp</button>
+          <button>Facebook</button>
+        </content>
+        <p class="text"><a href="#">Click for more diversity</a></p>
```

Comment:
While your current approach is not incorrect, I would recommend using an `icon` to improve accessibility.